tests/unit_node/http2_test.ts

```diff
@@ -165,3 +137,26 @@ Deno.test("[node/http2 client GET https://www.example.com]", async () => {
   assertEquals(status, 200);
   assert(chunk.length > 0);
 });
+
+Deno.test("[node/http2.createServer()]", async () => {
+  const server = http2.createServer((_req, res) => {
+    res.setHeader("Content-Type", "text/html");
+    res.setHeader("X-Foo", "bar");
+    res.writeHead(200, { "Content-Type": "text/plain; charset=utf-8" });
+    res.write("Hello, World!");
+    res.end();
+  });
+  server.listen(0);
+  const port = (<net.AddressInfo> server.address()).port;
+  const endpoint = `http://localhost:${port}`;
+
+  const response = await curlRequest([
+    endpoint,
+    "--http2-prior-knowledge",
+  ]);
+  assertEquals(response, "Hello, World!");
+  server.close();
+  // Wait to avoid leaking the timer from here
```

Comment:
Do you mind filing a bug against the sanitizers for this? We should not trigger the sanitizers if a timer fires in the next event loop.

Reply:
I've opened an issue here https://github.com/denoland/deno/issues/22764